sync-core/src/main/java/com/cloudant/sync/datastore/BasicDatastore.java

```diff
@@ -681,7 +702,7 @@ public BasicDocumentRevision call(SQLDatabase db) throws Exception {
 
                         long lineId = db.insert("localdocs", values);
                         if (lineId < 0) {
-                            throw new IllegalArgumentException("Can not insert new local doc, likely the docId exists already: "
+                            throw new DocumentException("Can not insert new local doc, likely the docId exists already: "
                                     + docId);
```

Comment:
Is this error even right? I don't think local docs are versions, at least not on Couch.